packages/sign-client/src/controllers/engine.ts

```diff
@@ -267,8 +270,7 @@ export class Engine extends IEngine {
       throw error;
     }
 
-    let { pairingTopic, proposer, requiredNamespaces, optionalNamespaces } = proposal;
-    pairingTopic = pairingTopic || "";
```

Comment:
Is this safe to remove in terms of backwards compat? Seems like existing (dapp) clients that don't do [this](https://github.com/WalletConnect/walletconnect-monorepo/pull/4553/files#diff-04e05e19b9270d1267b68d324e16997e4465d6879324fe4427a6194c00410eabR219) will have `pairingTopic: undefined` in the proposal they send, i.e. not a string?

Reply:
clients are handling the case even at the moment because native wallets do not provide the pairing topic anyway